docker/importer/importer.py

```diff
@@ -170,7 +183,11 @@ def run(self):
       if source_repo.name == 'oss-fuzz':
         self.process_oss_fuzz(source_repo)
 
-      self.process_updates(source_repo)
+      self.validate_source_repo(source_repo)
+      if not self._delete:
+        self.process_updates(source_repo)
+      if self._delete:
```

Comment:
nit: just if / else would simplify this.

```python
if self._delete:
  self.process_deletions(...)
else:
  self.process_updates(...)
```

Reply:
Simplified in 2abd779